### DISCUSSION TITLE
Does this work with Gravity Forms?

### DISCUSSION THREAD

**Discussion by stuartcusackie** [Interfacelab/ilab-media-tools#151] [Q&A]
Hi, Great plugin. It has been working flawlessly for us for a year now. 

I'm just wondering if there is a way to set it up so that a Gravity Form with a file upload field will automatically send a large file through this plugin to our digital ocean space?

Thanks.

**Comment by jawngee** [comment]
Depends, but generally no because gravity forms doesn't save uploads to the media library.  There may exist an addon for it that does though.

**Comment by jawngee** [comment]
Sorry I read this too fast.  No, you'd have to write your own add-on for gravity forms to do this.

The caveat to that is if an add-on existed that used the WordPress media uploader thing.  I know for WooCommerce vendor add-ons, this is what they use and Media Cloud direct uploads work in that situation.  But I don't know any gravity form add-ons that go that route.

**Reply by stuartcusackie** [reply]
I see I see. Thanks for the explanation! Yes, there doesn't seem to be anything like that for gravity forms, or for large file uploads with gravity forms. We will look into coding up something custom.

Thanks again!
